programs/clearing_house/src/controller/pnl.rs

```diff
@@ -124,3 +137,114 @@ pub fn settle_pnl(
 
     Ok(())
 }
+
+pub fn settle_expired_position(
```

Comment:
tests?

Reply:
check that user isnt being liquidated/bankrupt

Reply:
added checks and test for this